src/main/java/de/tum/in/www1/artemis/service/exam/monitoring/ExamActionService.java

```diff
@@ -0,0 +1,49 @@
+package de.tum.in.www1.artemis.service.exam.monitoring;
+
+import java.util.Collection;
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.domain.exam.monitoring.ExamAction;
+import de.tum.in.www1.artemis.repository.ExamActionRepository;
+
+@Service
+public class ExamActionService {
+
+    private final ExamActionRepository examActionRepository;
+
+    public ExamActionService(ExamActionRepository examActionRepository) {
+        this.examActionRepository = examActionRepository;
+    }
+
+    /**
+     * To avoid direct access to the {@link ExamActionRepository}, we use delegation save the {@link ExamAction}.
```

Comment:
Not critical: If you want to really use a javadoc for this simple method  (Why?), it makes more sense imo to explain what the method does and not why it is there. There are also seems to be a missing word: "we use delegation **to** save the..."

Reply:
Yes, I think comments are always useful :+1: and I think the name explains the function, so I just added an explanation to understand why we use it.

Reply:
What about the missing word in the comment?

Reply:
> Yes, I think comments are always useful 👍 and I think the name explains the function, so I just added an explanation to understand why we use it.

But do you not think it is a little bit much to write the exact same comment on every single service method you write?

Reply:
No imo its fine to be more detailed instead of less detailed. So i leave it as its :)